src/custom-elements/src/components/drawer/drawer.tsx

```diff
@@ -178,13 +180,32 @@ export class Drawer {
   }
 
   componentWillLoad(): void {
+    const target = this.el;
+    const config = {
+      attributes: true,
+      attributeOldValue: true,
+      attributeFilter: ['title']
+    };
+
+    const subscriberCallback = (mutations) => {
+      mutations.forEach((mutation) => {
+        this.drawerTitle = mutation.target.title;
```

Comment:
Please, update the value only if content is different to reduce unwanted re-rendering.

Reply:
Please, don't forget to remove any listener attached. I.e. don't forget to call [observer.disconnect](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver/disconnect) on [componentDidUnload()](https://stenciljs.com/docs/component-lifecycle). Optionally, a better approach could be to do the `observe` and `disconnect` in the [connected](https://stenciljs.com/docs/component-lifecycle#connectedcallback-) and [disconnected](https://stenciljs.com/docs/component-lifecycle#disconnectedcallback-) callbacks.